packages/extension-sample/src/setup-server.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* istanbul ignore next */
+export default ({app}) => {
+    console.log('Setup Server App')
+}
```

Comment:
do we want to make the server extension a JS class too?

Reply:
Yes. I briefly talked about wanting to do this in the kick off meeting last week and how we'd want symmetry with those APIs. That will be done in another PR tho.

Reply:
👍